internal/gojs/fs.go

```diff
@@ -464,16 +464,16 @@ type jsfsFchmod struct{}
 
 func (jsfsFchmod) invoke(ctx context.Context, mod api.Module, args ...interface{}) (interface{}, error) {
 	fd := goos.ValueToInt32(args[0])
-	mode := custom.FromJsMode(goos.ValueToUint32(args[1]), 0)
+	_ /* mode */ = args[1]
```

Comment:
I'll be the nitpicker here; as I am seeing this is repeated I assume it's intended: is it a common style? I would normally use

```
_ = args[1] // mode
```

(same for the rest of the file)

Reply:
works for me!